tests/test_parser.py

```diff
@@ -214,3 +214,58 @@ def test_parser_reserved():
     all_metrics = parser.parse_metrics(contents, {'allow_reserved': True})
     errors = list(all_metrics)
     assert len(errors) == 0
+
+
+def test_parser_slash_in_name():
```

Comment:
This looks great! I wonder if we should do the same for the `#` character, which we're using as a separator in glean [here](https://github.com/mozilla-mobile/android-components/blob/b83fae2c1b1a444b996d31a5b05d1f73737a004e/components/service/glean/src/main/java/mozilla/components/service/glean/storages/GenericScalarStorageEngine.kt#L115). What do you think?

Reply:
That's a good idea.  I've updated this to test for more cases like this.